### PR TITLE
Improved Email templates

### PR DESCRIPTION
**Pull request by KiraboIsaacBuyondo**
# Description

- This PR aims to improve the anonymous_user_to_project email using the standard emailing template created in Figma. It takes care of the standard branding, sizes, font styles, general layout, and background formats.

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Trello Ticket ID

1. https://trello.com/c/IWyB6AVy  
2. https://trello.com/c/IQqVNWgZ

## How Can This Be Tested?

- Run the branch on your local machine. 
- Send an invitation email to an anonymous user.
- Check its appearance on various devices for font size, colors, etc.
- Test the social media icons for Medium, LinkedIn, YouTube, and X to ensure they link to the official Crane Cloud social media platforms.

## Screenshot
Crane Cloud

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: templates/user/anonymous_user_to_project.html [hunk 1]
```diff
@@ -1,252 +1,129 @@
...
-                  font-size: 18px;
-                  font-weight: 400;
-                  line-height: 25px;
-                "
-              >
-                <p style="margin: 0">Hey there,</p>
-              </td>
-            </tr>
-            <tr>
-              {% if success ==true %}
```
Comment (Mubangizi) [inline templates/user/anonymous_user_to_project.html:148]: As you can see `success` is for inviting users for collaboration, meaning you will break this workflow if you just eliminate it

File: templates/user/user_to_project.html [hunk 2]
```diff
@@ -1,251 +1,130 @@
...
-                  font-size: 18px;
-                  font-weight: 400;
-                  line-height: 25px;
-                "
-              >
-                <p style="margin: 0">Hi {{name}},</p>
-              </td>
-            </tr>
-            <tr>
-              {% if success ==true %}
```
Comment (Mubangizi) [inline templates/user/user_to_project.html:148]: You can also see `success` is used for informing users what they have been assigned as in a project

**Review by Mubangizi — CHANGES_REQUESTED** [review]
Great work Kirabo, just a little feedback.
1. Please put all the content in the white box, ie the greeting and the contact information. Also work on making the contact info ie footer areas font slightly smaller than the general body font. this is in accordance to the image of the email shared
2. Remove the **icons** folder and its files, then use a library ie `font Awesome` to add the socials icons. something like

> ```
>
><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font->awesome/4.7.0/css/font-awesome.min.css">
>
>
><a href="#" class="fa fa-facebook"></a>
><a href="#" class="fa fa-twitter"></a>
>```

**Review by Mubangizi — APPROVED** [review]
LGTM

**Review by rhodinemma — APPROVED** [review]
Good work on this @KiraboIsaacBuyondo

**Pull request merged**